client/tests/unit/reserve-submit.spec.ts

```diff
@@ -10,6 +10,9 @@ const vuetify = new Vuetify()
 
 localVue.use(Vuetify)
 
+/* commented out lines and blocks of tests were correct and passing before it was decided not to allow auto
```

Comment:
+1 for the comment!

But once this is merged, it's going to be hard to know that this comment applies to some changes below. For this reason I recommend moving the comment to the subject code blocks.

Reply:
Or maybe define a global prop to toggle between old/new code?